src/main/java/org/geppetto/frontend/controllers/ConnectionHandler.java

```diff
@@ -361,38 +361,30 @@ public void runExperiment(String requestID, long experimentID, long projectId)
 
 		IGeppettoProject geppettoProject = retrieveGeppettoProject(projectId);
 		IExperiment experiment = retrieveExperiment(experimentID, geppettoProject);
-		if(geppettoProject.isVolatile())
-		{
-			info(requestID, Resources.VOLATILE_PROJECT.toString());
-			return;
-		}
-		else
+		try
 		{
-			try
+			// run the matched experiment
+			if(experiment != null)
 			{
-				// run the matched experiment
-				if(experiment != null)
-				{
-					// TODO: If experiment is in ERROR state, user won't be able
-					// to run again.
-					// We reset it to DESIGN to allow user to run it for second
-					// time
-					if(experiment.getStatus() == ExperimentStatus.ERROR)
-					{
-						experiment.setStatus(ExperimentStatus.DESIGN);
-					}
-					geppettoManager.runExperiment(requestID, experiment);
-				}
-				else
+				// TODO: If experiment is in ERROR state, user won't be able
+				// to run again.
+				// We reset it to DESIGN to allow user to run it for second
+				// time
+				if(experiment.getStatus() == ExperimentStatus.ERROR)
 				{
-					error(null, "Error running experiment, the experiment " + experimentID + " was not found in project " + projectId);
+					experiment.setStatus(ExperimentStatus.DESIGN);
```

Comment:
@gidili This wasn't part of the code I added for this branch, got added after merging development into here. The reason behind it though it's because if an experiment fails, the idea was to set it back to design to allow it to be re-run, otherwise it would be stuck on ERROR and the user wouldn't be able to run it again

Reply:
@jrmartin I think it was by design that if it errors you have to clone it and tweak stuff and run again, this change might have implications we are not thinking about... for example if it's back to design it could hide the error for the user (while right now once it's no error you can see the error). Let's discuss with @tarelli at the geppetto meeting though.